### ISSUE TITLE
MatTable page size not stable after updating 1.7.4

### ISSUE THREAD

**Issue by novfal** [SamProf/MatBlazor#247]
Page size automatically(zigzag) change 10 to 5

**Comment by benishue** [comment]
Did you try to set the following properties?

_PagingDataPropertyName="Id" PagingRecordsCountPropertyName="Id" PageSize="3"_

For me it's working

**Comment by novfal** [comment]
try this after that also same problem

**Comment by novfal** [comment]
@SamProf   table page size changed automatically from 10(fixed page size) to 5 and also in mat tab selection values not loaded i use the button for that click

**Comment by coffeelake** [comment]
I also have problems with PageSize, it flicker between my PageSize and default value.
When i load the page for the first time its allways default PageSize and as soon as i do something(push button or something) it jumps to my PageSize value.

**Comment by novfal** [comment]
in matblazor 1.7.2 working fine

**Comment by vanderstack** [comment]
I was able to temporarily workaround this by invoking StateHasChanged within the lifecycle method OnAfterRenderAsync, although this momentarily flashes the widget with fewer rows than I intended. To avoid this I put the widget behind an if block which is toggled during onAfterRender.

Looking forward to this being officially fixed, I've just recently discovered this library and very much enjoy and appreciate it.
